### PR TITLE
Reexport Observables macros

### PR DESCRIPTION
**Pull request by asinghvi17**
cc @jkrumbiegel

### CODE REVIEW

File: src/AbstractPlotting.jl [hunk 1]
```diff
@@ -1,4 +1,17 @@
 module AbstractPlotting
+                                                                                                                                                          5.11G   12:51:45 
+################################################################################
```
Comment (SimonDanisch) [inline src/AbstractPlotting.jl:3]: what this above?

File: src/AbstractPlotting.jl [hunk 2]
```diff
@@ -13,7 +13,13 @@ using Markdown, DocStringExtensions
 # serialize events
 using Serialization 
 
-
+using FFMPEG # get FFMPEG on any system!
+using Observables, GeometryTypes, StaticArrays, IntervalSets, PlotUtils
+using ColorBrewer, ColorTypes, Colors, ColorSchemes
+using FixedPointNumbers, Packing, SignedDistanceFields
+using Markdown, DocStringExtensions # documentation
+using Serialization # serialize events
```
Comment (mschauer) [inline src/AbstractPlotting.jl]:
```suggestion
```

File: src/AbstractPlotting.jl (hunk 1 above)
Comment (asinghvi17) [reply src/AbstractPlotting.jl:3]: Comment box...I just put it there for some structure.

Comment (SimonDanisch) [reply src/AbstractPlotting.jl:3]: Yeah, but there is some weird unicode above it?

Comment (asinghvi17) [reply src/AbstractPlotting.jl:3]: huh, that's really weird.  Let me check tomorrow...

**Comment by SimonDanisch** [comment]
Closing this for nowm until it gets picked up again ;)

**Pull request closed without merging**